docs/outputs.md

```diff
@@ -32,6 +32,22 @@ be used to set the compression level (between 1 and 9, default is 5). Compressio
 altogether with the CMake build option `PARTHENON_DISABLE_HDF5_COMPRESSION`. See the [build
 doc](building.md) for more details.
 
+## Tuning HDF5 Performance
+
+Tuning IO parameters can be passed to Parthenon through the use of environment variables. Available environment variables are:
+
+|  Environment Variable | Default | Value | Description |
+|---|---|---|---|
+| H5_sieve_buf_size | disabled | int | Sets the maximum size of the data sieve buffer, in bytes. The value should be equal to a multiple of the disk block size. If no value is set then the default is 256 KiB. |
+| H5_meta_block_size | enabled | int | Sets the minimum metadata block size, in bytes. If no value is   set then the default is 16 MiB. |
+| H5_alignment_threshold | disabled | int | The threshold value, in bytes, of H5Pset_alignment. Setting to 0 forces  everything to be aligned. If a value is not set then the default is 1 MiB. Setting the environment variable automatically enables alignment. |
+| H5_alignment_alignment | disabled | int | The alignment value, in bytes, of H5Pset_alignment. If a value is not set then the default is 16 MiB.  Setting the environment variable automatically enables alignment.  H5Pset_alignment sets the alignment properties of a file access property list. Choose an alignment  that is a multiple of the disk block size, enabling this usually shows better performance on GPFS.   Enabling may increase the file size significantly. |
+| H5_defer_metadata_flush | disabled | int | Value of 1 enables deferring metadata flush. Value of 0 disables. |
+| MPI_access_style | enabled | string | Specifies the manner in which the file will be accessed until the file is closed. Default is "write_once" |
+| MPI_collective_buffering | disabled | int | Value of 1 enables MPI collective buffering. Value of 0 disables. |
+| MPI_cb_block_size | N/A | int | Sets the block size, in bytes, to be used for collective buffering file access. Default is 1 MiB. |
+| MPI_cb_buffer_size | N/A | int | Sets the total buffer space, in bytes, that can be used for collective buffering on each target node,  usually a multiple of cb_block_size. Default is 4 MiB. |
```

Comment:
This added documentation is great! I notice there's no comment on how these options effect performance. I suppose it is simply problem and hardware dependent?

Reply:
> I wonder if we could include some of the results of your analysis on what these options do in the wiki or documentation? Would that be okay?

Yes, that would be okay.

Reply:
Great! :+1:

Reply:
> This added documentation is great! I notice there's no comment on how these options effect performance. I suppose it is simply problem and hardware dependent?

It would be system and hardware-dependent, one thing I hope this PR does is to make exploring how the different settings would affect performance. But some general advice for setting them can be given.

I left some of the tuning parameters enabled in the original code as I'm not sure that maybe there is a use case that they are beneficial.

Reply:
Got it. That makes sense.

Reply:
I've done some more test runs with them enabled and disabled for some of them, and I'll add them to the report.

Reply:
I changed most of the options to be disabled to be on the safe side, although set_aligment is usually suitable for good performance, so if the file size is still reasonable after enabling, this can be enabled by default.